lib/CMakeLists.txt

```diff
@@ -190,11 +190,12 @@ elseif ("${CMAKE_SYSTEM_NAME}" MATCHES "Linux")
     )
 elseif ("${CMAKE_SYSTEM_NAME}" MATCHES "FreeBSD")
     set(LIBFACTER_PLATFORM_SOURCES
-        "src/facts/bsd/collection.cc"
```

Comment:
I think it would still theoritically be used for DragonflyBSD, NetBSD or other BSDs if people were to build Facter 3.X? Should I still remove?